token/cli/src/main.rs

```diff
@@ -1375,15 +1376,23 @@ fn command_close(
             )
             .into());
         }
+
+        let recipient_account = config.rpc_client.get_token_account(&recipient)?;
+        is_recipient_wrapped = recipient_account.is_some() && recipient_account.unwrap().is_native;
```

Comment:
nit for more functional clarity: 
```suggestion
        is_recipient_wrapped = recipient_account.and_then(|x| x.is_native).unwrap_or(false)
```